### PR TITLE
Added info about We did not receive the submission

### PR DESCRIPTION
**Pull request by dhairyya**
Tried to call out customer gaps which is mostly leading this error condition and thus escalation to us.

### CODE REVIEW

**Comment by dhairyya** [comment]
@chrisda could you please approve this for me?

File: microsoft-365/security/office-365-security/submissions-error-messages.md [hunk 1]
```diff
@@ -40,3 +40,14 @@ If you encounter this error message, then either of the following conditions mig
 - You tried to submit an email message that was filtered by EOP or Defender for Office 365, but we're still in the process of collecting the required metadata (descriptive data) about the message.
 
   If you wait "a while" and submit the message again, the submission will be successful.
+  
+  
+## We did not receive the submission, please fix the problem and resubmit
+
+If you encounter this error message, then either of the following conditions have occured:
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/submissions-error-messages.md]:
```suggestion
If you encounter this error message, then either of the following conditions has occurred:
```

File: microsoft-365/security/office-365-security/submissions-error-messages.md [hunk 2]
```diff
@@ -40,3 +40,14 @@ If you encounter this error message, then either of the following conditions mig
...
+
+- You are trying to submit an email which has been deleted or is no longer in the mailbox or quarantine.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/submissions-error-messages.md]:
```suggestion
- You are trying to submit an email that has been deleted or is no longer in the mailbox or quarantine.
```

File: microsoft-365/security/office-365-security/submissions-error-messages.md [hunk 3]
```diff
@@ -40,3 +40,14 @@ If you encounter this error message, then either of the following conditions mig
...
+
+- You have Exchange Transport Rules (ETR) or Connectors or Data Loss Prevention (DLP) rules preventing the message from coming to us
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/submissions-error-messages.md]:
```suggestion
- You have Exchange Transport Rules (ETR), connectors or Data Loss Prevention (DLP) rules preventing the message from reaching us. 
```

File: microsoft-365/security/office-365-security/submissions-error-messages.md [hunk 4]
```diff
@@ -40,3 +40,14 @@ If you encounter this error message, then either of the following conditions mig
...
+  
+## We did not receive the submission, please fix the problem and resubmit
+
+If you encounter this error message, then either of the following conditions have occured:
+
+- You are trying to submit an email which has been deleted or is no longer in the mailbox or quarantine.
+
+- You have Exchange Transport Rules (ETR) or Connectors or Data Loss Prevention (DLP) rules preventing the message from coming to us 
+
+So please make sure to check that both of these conditions are false before submitting the message again.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/submissions-error-messages.md]:
```suggestion
Be sure to check that both of these conditions are false before submitting the message again. 
```

**Pull request merged**